package.json

```diff
@@ -26,11 +26,9 @@
   },
   "homepage": "https://github.com/TimDaub/better-sqlite3-session-store#readme",
   "devDependencies": {
-    "ava": "3.13.0",
-    "better-sqlite3": "7.1.1",
-    "express-session": "1.17.1"
-  },
-  "dependencies": {
-    "date-fns": "2.16.1"
+    "ava": "^4.3.3",
```

Comment:
as a feedback, you should never "just" upgrade a major version number as it contains breaking changes.

Reply:
here's the fix for it: 86a4d14